pages/docs/tutorials/streetlights.md

```diff
@@ -224,7 +224,7 @@ EOT`}
 
 You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
 
-We would love to see what you create with AsyncAPI! As an open-source project, we're open to proposals, questions, suggestions, and contributions. If you don't feel in the mood to contribute but you're using AsyncAPI, just raise your hand by [creating an issue in our Github repo](https://github.com/asyncapi/asyncapi/issues/new) or [join our Slack channel](https://www.asyncapi.com/slack-invite/). Don't be shy. :simple_smile:
```

Comment:
@pratik2315 @alequetzalli What if instead of adding the emoji-like `:simple_smile:`, we directly add the emoji 😃 in the markdown file. See the example used in GSoD blog written [here](https://github.com/asyncapi/website/blob/master/pages/blog/google-season-of-docs-2022.md?plain=1#L23).

Reply:
Hey @akshatnema If im not wrong they did add emoji like "😃" Like but it got converted to :simple_simple: after the deployment?

Reply:
Like i dont see any reason they will comment out an emoji😆 i may be wrong!

Reply:
Nope, they added the classic emoji-style (`:simple_smile:`) used in the markdown to add an emoji, but what I recommend you to use it as a symbol in the markdown file.

Reply:
also tried out your just now! it works!! the original writer must've used :simple_simple: word by word instead of emoji and forgotten about it in deployment!

Reply:
But on a side note @akshatnema what do u think about using emoji's in a documentation? I have read lots of docs but I generally don't find any emoji's in it? what are your thoughts, would love to hear it! also @alequetzalli tell me what do you think about it!

Reply:
@pratik2315 As mentioned before, we can consider adding more emojis in the Docs, but we would need to start a separate discussion on this, and I'd have to think more about what emojis make sense to use and where. :)  This is basically a style guide item that would need planning too. 

A blog post is informal, so seeing emojis there is not the same example as for Docs. 

As discussed, I think it makes sense to remove it from this PR. But if you really think this emoji helps you as a contributor, you can go ahead and exchange `:simple_smile:` for the actual emoji to that the rendering issue is resolved.  :)

Reply:
Got it! About the emoji I think we should remove emoji as of now. As you said, this style guide item needs planning. So removing the emoji for now makes perfect sense to me👍

Reply:
@pratik2315 And another example of a contribution idea here is that since you are interested in the question of emojis in Documentation, you are totally free to open a new documentation issue to promote planning emojis in the style guide. This way more people can become aware and vote on the idea as a community as a whole, and you still get to look into emojis in Docs. (If you're really curious of course!)

Reply:
Hmm. Seems interesting! 😃🔥